src/uct/ib/base/ib_device.h

```diff
@@ -212,6 +212,7 @@ typedef struct uct_ib_device {
     uct_ib_device_attr          dev_attr;        /* Cached device attributes */
     uint8_t                     first_port;      /* Number of first port (usually 1) */
     uint8_t                     num_ports;       /* Amount of physical ports */
+    uint8_t                     port_select_mode; /* lag device port select mode */
```

Comment:
why not `uct_dc_lag_port_select_t`?

Reply:
@evgeny-leksikov do you mean define a new enum type?

Reply:
@evgeny-leksikov It may not proper to do that. According to ```LAG CONTEXT Field Descriptions```, there're three modes for ```port_select_mode```. It's not aligned with ```uct_dc_lag_port_select_t``` defined in this PR.

Reply:
Done